### PR TITLE
Update workshop enrollment receipt contact information

### PR DESCRIPTION
**Pull request by joshlory**
Address https://github.com/code-dot-org/code-dot-org/pull/25973#issuecomment-438394274.

### CODE REVIEW

File: dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml [hunk 1]
```diff
@@ -22,3 +22,12 @@
     %td
       = @workshop.organizer.name
       (#{mail_to(@workshop.organizer.email)})
+  - unless @workshop.facilitators.empty?
+    %tr
+      %td.heading
+        Facilitator(s):
```
Comment (mehalshah) [inline dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml]: Suggest
`= @workshop.facilitators.count == 1 ? 'Facilitator:' : 'Facilitators'`

File: dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml [hunk 2]
```diff
@@ -22,3 +22,12 @@
     %td
       = @workshop.organizer.name
       (#{mail_to(@workshop.organizer.email)})
+  - unless @workshop.facilitators.empty?
```
Comment (mehalshah) [inline dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml]: Prefer `if @workshop.facilitators.any?` to avoid the double negative

**Review by mehalshah — APPROVED** [review]

File: dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml (hunk 1 above)
Comment (joshlory) [reply dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml]:
```ruby
0> @workshop.facilitators.length
=> 1

0> @workshop.facilitators.count
=> 0
```
🤔

Comment (joshlory) [reply dashboard/app/views/pd/workshop_mailer/_workshop_logistics.html.haml]: This is on the `/rails/mailers/pd/workshop_mailer/teacher_enrollment_receipt__csf` view.

**Pull request merged**